tests/jballerina-unit-test/src/test/resources/test-src/types/string/bstring-table-test.bal

```diff
@@ -20,14 +20,11 @@ type Employee record {
     float salary;
 };
 function testTableGeneration() returns int {
-   table<Employee> tbEmployee = table {
-           {key id, name, salary},
-           [
-               {1, "Mary🤒", 300.5},
-               {2, "John💉", 200.5},
-               {3, "Jim", 330.5}
-           ]
-       };
+   table<Employee> tbEmployee = table [
+               {id: 1, name: "Mary", salary: 300.5},
+               {id: 2, name: "John", salary: 200.5},
+               {id: 3, name: "Jim", salary: 330.5}
```

Comment:
Why are we changing these test cases?

Reply:
Test cases are the same. Syntax of initializing the table got changed.

Reply:
"🤒", "💉" are missing in the updated one

Reply:
Didn't know they were intentionally added hence removed. Added back in 70e13a6 .